sdk/iOS/test/MSSyncTableTests.m

```diff
@@ -2184,6 +2258,7 @@ -(void) testPurgePendingOpsOnDifferentTableSuccess
             
             done = YES;
         }];
+		XCTAssertNotNil(purge);
```

Comment:
I had this issue with another PR in the past. I use tab indentation, so lays out correctly in Xcode but GitHub adds the extra indentation when displaying it in the diff (maybe due to the plus symbol next to it)